packages/components/psammead-image/README.md

```diff
@@ -59,12 +59,17 @@ const WrappingContainer = ({ alt, src, height, width }) => (
 | Prop     | Type          | Required | Default | Example                                                                                                |
 | :------- | :------------ | :------- | :------ | :----------------------------------------------------------------------------------------------------- |
 | `alt`    | string        | Yes      | -       | "A picture of a cat"                                                                                   |
-| `height` | number/string | Yes      | null    | 450                                                                                                    |
+| `height` | number/string | No       | null    | 450                                                                                                    |
 | `src`    | string        | Yes      | -       | "https://bbc.com/300/cat.jpg"                                                                          |
 | `srcset` | string        | No       | null    | "https://bbc.com/300/cat.jpg 300w, https://bbc.com/450/cat.jpg 450w, https://bbc.com/600/cat.jpg 600w" |
-| `width`  | number/string | Yes      | -       | 600                                                                                                    |
+| `width`  | number/string | No       | null    | 600                                                                                                    |
+
+The `height` and `width` props are optional, in some cases to preserve the image ratio you might specify either `height` or `width` and let the browser scale the image accordingly.
+
+However when not specified the browser will not be able to determine the size of the image, the browser will therefore build the page twice or more depending on the number of images you have. First build is for the browser to display all the text and once the image is downloaded and size determined a second build to wrap the texts around the image.
```

Comment:
By build, do you mean paint?

Reply:
Yah paint. Am changing it to `paint` for clarity. Thanks for bringing that up.